buddyboss/buddyboss-platform
# Visual issues in admin with different inputs

### All Submissions:


Fixes #738 .

### How to test the changes in this Pull Request:

1. Go to Dashboard
2. Edit User Page
3. Go to Extended Profile Tab, Check the inputs alignment

### Proof Screenshots or Video

https://prnt.sc/rydn7y
https://prnt.sc/rydo69

### Other information:
